osu.Game.Rulesets.Catch/Edit/CatchHitObjectComposer.cs

```diff
@@ -77,8 +88,27 @@ protected override void Update()
             updateDistanceSnapGrid();
         }
 
+        public override bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
+        {
+            switch (e.Action)
+            {
+                case GlobalAction.IncreaseScrollSpeed:
+                    this.TransformBindableTo(timeRangeMultiplier, timeRangeMultiplier.Value - 1, 200, Easing.OutQuint);
+                    break;
+
+                case GlobalAction.DecreaseScrollSpeed:
+                    this.TransformBindableTo(timeRangeMultiplier, timeRangeMultiplier.Value + 1, 200, Easing.OutQuint);
+                    break;
```

Comment:
Hmm, these default to F3/F4 don't they? That will conflict with the editor if so.

Also I can't seem to get this to work, which keys were you testing with?

Would be nice to also have a way to adjust via the editor menu or otherwise.

Reply:
That was mentioned in the OP. For what it's worth, every time I play mania I keep trying to use Ctrl+(+)/(-) to adjust scroll, but that changes UI scale instead.

I tested it with Ctrl+1/2 after rebinding. Agree with being adjustable via editor menu.

Reply:
Yeah I saw the mention after posting my comment.

I was trying various keys but the editor really does take control of a lot which makes it hard to find something good to bind it to.

Reply:
Editor menu is not currently customizable by rulesets. I think I want to it leave as is for this PR.

Reply:
Sure, we can merge it, but it's not going to be usable (or visible) to most users as is. I'll add an inline comment for future visitors.